### PR TITLE
MGMT-1118 Update ignition spec version for RHCOS

### PR DESCRIPTION
**Pull request by rollandf**

### CODE REVIEW

**Comment by rollandf** [comment]
To merge only when the image will be updated to rhcos in Image Builder

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ type Config struct {
 }
 
 const ignitionConfigFormat = `{
-"ignition": { "version": "3.0.0" },
+"ignition": { "version": "2.2.0" },
```
Comment (eranco74) [inline internal/bminventory/inventory.go:88]: :fearful: 
:crossed_fingers:

File: internal/bminventory/inventory.go [hunk 2]
```diff
@@ -103,6 +103,7 @@ const ignitionConfigFormat = `{
 },
 "storage": {
     "files": [{
+	  "filesystem": "root",
```
Comment (eranco74) [inline internal/bminventory/inventory.go:106]: Fix indentation

Comment (rollandf) [reply internal/bminventory/inventory.go]: Done

**Pull request merged**